### DISCUSSION TITLE
This projet but with chatgpt

### DISCUSSION THREAD

**Discussion by pascalandy** [imartinez/privateGPT#399] [General]
I believe there might be similar solutions out there, although I'm not certain. However, based on my experience, the other language models I've tried don't quite measure up to the capabilities of ChatGPT.

**Comment by KeulzZz** [comment]
But chatGPT can't be run locally. That being said, i believe code interpreter of chatgpt should be able to do similar stuff.

**Comment by hcchengithub** [comment]
I don't care exposing sensitive data to OpenAI. How to modify .env to choose ChatGPT-4 as the AI for privateGPT?

**Reply by vsingh-healthians** [reply]
you can get the open ai api key!

**Comment by tompayne36** [comment]
I'd be interested in this as well!

**Comment by hcchengithub** [comment]
This is the answer: https://github.com/imartinez/privateGPT/issues/813

**Comment by anoop22** [comment]
The answer referred above was for older model i believe that is GPT 3. using gpt 3.5 calls for a different langchain function and style of code

**Comment by phatal** [comment]
If we can add interaction through ChatGPT models with implementation in Langchain, so there definitely should be a way to implement PrivateGPT's killer features to ChatGPT itself (with a browser extension or something else), because interaction through the API is happening anyway with a remote server.

Because poor size of context window nowaday one of the most annoying problems in interacting with chatgpt
